activity_calendar/feeds.py

```diff
@@ -110,7 +109,9 @@ def item_timestamp(self, item):
 
     def item_link(self, item):
         # The local url to the activity
-        return item.get_absolute_url()
+        # Because of the repetitition of the activity there is not 1 date. So instead let's just reroute it to the
+        # app activity page for now
+        return "https://app.kotkt.nl/api/calendar/fullcalendar"
```

Comment:
Hardcoded URL. You can try [this](https://docs.djangoproject.com/en/3.1/ref/request-response/#django.http.HttpRequest.build_absolute_uri) instead

Reply:
It'll be changed in a later PR that addresses feeds. This is a quick patch. Besides, from what I can tell Google doesn't use the link anyway